recommenders/popularity_recommender.py

```diff
@@ -1,31 +1,27 @@
-from recommenders import data_movies, data_ratings, default_poster_url
-import pandas as pd
-
-
-def recommend_popular_movies():
-
```

Comment:
The variable 'movies_ratings_merged' should be indented to align with the starting point of the line above it.